docs/csharp/programming-guide/classes-and-structs/index.md

```diff
@@ -11,101 +11,99 @@ helpviewer_keywords:
   - "C# language, classes"
 ms.assetid: cc39dbda-8754-423e-b5b1-16a1db0734c0
 ---
-# Classes and structs (C# programming guide)
+# Classes, structs, and records (C# programming guide)
```

Comment:
- metadata wasn't updated.
- Should the file be renamed and have a redirection? or it's okay to leave it as is?

Reply:
* I'm updating the metadata
* File name OK to leave as-is